ensime_shared/ensime.py

```diff
@@ -279,11 +336,15 @@ def au_buf_leave(self, client, filename):
 
     @execute_with_client()
     def au_cursor_hold(self, client, filename):
-        client.on_cursor_hold(filename)
+        """Handler for CursorHold autocommand event."""
+        self.watchdog_client_server_connection(client)
+        client.unqueue_and_display(filename)
+        client.editor.cursorhold()
```

Comment:
This call to editor's `cursorhold` causes the event to be retriggered every second when no activity is happening. I don't think this is the intended behaviour.